app/src/main/java/de/westnordost/streetcomplete/map/components/TracksMapComponent.kt

```diff
@@ -21,47 +21,49 @@ class TracksMapComponent(ctrl: KtMapController) {
     private val layer2 = ctrl.addDataLayer(LAYER2)
 
     private var index = 0
-    private var tracks: MutableList<MutableList<LngLat>>
+    private class Track(val trackpoints: MutableList<LngLat>, val isRecording: Boolean)
```

Comment:
Could be a `data class` right? Also, maybe another name would be better to avoid confusion with the `Track` class from `app/src/main/java/de/westnordost/streetcomplete/data/osmtracks/Track.kt`.

Reply:
I changed it to a `data class`  not sure of a better name currently.